tests/idris2/pkg/multiline-comments/test.ipkg

```diff
@@ -0,0 +1,7 @@
+package test
+version = 0.1.0
+authors = "Boyd Stephen Smith Jr."
+main = Main
+executable = "test"
+{- "." is the top-level package directory -}
```

Comment:
May as well throw in a multi-line comment that is more than one line long (in addition to this comment).

Reply:
Done.